000-HowToHack/WTH-ChallengeZero-Template.md

```diff
@@ -0,0 +1,108 @@
+# Challenge 00 - Prerequisites - Ready, Set, GO!
+
+<!-- REMOVE_ME ${navigationLine} (remove this from your MD files if you are writing them manually, this is for the automation script) REMOVE_ME -->
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+<!-- If you are using and editing this template manually, ensure the navigation link below is updated to link to next challenge relative to the current challenge. The "Home" link should always link to the homepage of the hack which is the README.md in the hack's parent directory. -->
+**[Home](../README.md)** - [Next Challenge >](./Challenge-01.md)
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+***This is a template for "Challenge Zero" which focuses on getting prerequisites set up for the hack. The italicized text provides hints & examples of what should or should NOT go in each section.***
+
+***We have included links to some common What The Hack pre-reqs in this template. All common prerequisite links go to the WTH-CommonPrerequisites page where the are more details on what each tool's purpose is.***
```

Comment:
should say "where there are more details..."

Reply:
done.